### DISCUSSION TITLE
Incoming Mail Help

### DISCUSSION THREAD

**Discussion by Xteri08** [postalserver/postal#2825] [Help with using Postal]
Hey there i just managed to install postal in arm64/v8. So i got this question how can i setup incoming mail. I didnt find any proper doc related to it. Thanks for your time

**Comment by willpower232** [comment]
What are you trying to set up specifically?

**Reply by Xteri08** [reply]
I am trying to setup incoming mail.
Like I got contact@mydomain.com then I am able to send outgoing mail but if I receive a mail it won't work so how will I make that functional

**Reply by willpower232** [reply]
Postal has no concept of mailboxes so you would have to set up a route which delivered incoming mail to your application

**Reply by Xteri08** [reply]
what mailboxe would you recommend to run along side postal in same vps

**Reply by willpower232** [reply]
Personally I wouldn't, I hate hosting email inboxes 😅 

Postals main job is helping things like websites, servers, and maybe printers send email messages.

If you want a traditional mailbox, you probably want something that isn't Postal.

**Reply by Xteri08** [reply]
i really like how postal got a clean gui and stuff for smtp. Thats the only reason i want it but i am a bit stuck at some edges i do have a discussion open related to it thanks for the response here.
